pkgs/by-name/im/imhex/package.nix

```diff
@@ -46,20 +59,34 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
     owner = "WerWolv";
     repo = "ImHex";
-    rev = "refs/tags/v${version}";
+    rev = "refs/tags/v${finalAttrs.version}";
     hash = "sha256-6QpmFkSMQpGlEzo7BHZn20c+q8CTDUB4yO87wMU5JT4=";
   };
 
-  nativeBuildInputs = [
-    autoPatchelfHook
-    cmake
-    llvm
-    python3
-    perl
-    pkg-config
-    rsync
+  patches = [
+    (fetchpatch {
```

Comment:
A commit why this patch is necessary and whether it can be removed with the next or future update would be helpful.